wpeverest/everest-forms
# Enhancement - Minimum length feature on the Paragraph/Single Line Text field.

### All Submissions:



### Changes proposed in this Pull Request:

Minimum length feature added on the paragraph/single-line text field.

### How to test the changes in this Pull Request:

1. Create a form with fields paragraph and/or single line text field.
2. go to Advanced Options of Field Options of these fields and enable Minimum Length.
3. Verify, the minimum length feature works properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Minimum length feature on the Paragraph/Single Line Text field.
